### DISCUSSION TITLE
shapes.txt for Elbląg?

### DISCUSSION THREAD

**Discussion by chrissz2613** [transitland/transitland#447] [Q&A]
I had a question regarding the GTFS dataset for the Polish city of Elbląg.

https://www.transit.land/feeds/f-u3tu-elblag

Do you know if a shapes.txt exists for this transit network? I didn't see it in the downloadable dataset. I wasn't sure if I was missing something.

**Comment by drewda** [comment]
No, that feed does not includes a `shapes.txt` file.

Therefore, the Transitland platform creates stop-to-stop geometries that are just straight lines.

You could try contacting the feed creators at https://przyjazdy.pl/english
